kubeflow/trainer/rhai/jit_checkpoint_code.py

```
@@ -0,0 +1,204 @@
+        """Check if a checkpoint has been requested."""
+        return self.checkpoint_requested
+
+
+    class JITCheckpointCallback(TrainerCallback):
+        """Transformers callback that integrates JIT checkpointing with trainer lifecycle."""
+
+        def __init__(self):
+            self.jit_manager = None
+            self._trainer_ref = None
+
+        def on_train_begin(self, args, state, control, **kwargs):
```

Ruff (0.14.5)

Unused method argument: `args`

(ARG002)